### ISSUE TITLE
Horizontal lanes

### ISSUE THREAD

**Issue by guillegette** [rcdexta/react-trello#220]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Horizontal lanes, will be an extra layer of organization to add to the board. For example: you can have categories for the cards, or who is assigned, etc. 

**Describe alternatives you've considered**
This could be solved putting 2 boards together one after the other one, but the idea will be to DnD cards from one horizontal lane to another, be able to DnD the columns like we do today, etc so 2 board will not look very pretty.

**Additional context**
![board2](https://user-images.githubusercontent.com/938027/58520536-1f6a7680-816d-11e9-938b-f27cb9bc5a66.png)

**Comment by Kalaschnik** [comment]
You can achieve this by loading multiple boards with different data:
https://rcdexta.com/react-trello/?selectedKind=Multiple%20Boards&selectedStory=Two%20Boards&full=0&addons=0&stories=1&panelRight=0

**Comment by guillegette** [comment]
Thanks @Kalaschnik I mentioned that as an alternative I have considered. The problem is that you would not be able to move cards between boards, moving one column will only move in 1 board and not all of the boards, creating a new column will not be reflected in the rest and many more events.

I am willing to put the hours into this if there is enough interest.

**Comment by Kalaschnik** [comment]
That is true. We actually use multiple lanes/boards (lanes in their original meaning, see #126) ending up with a data structure like this:

```
{
  "swim-lanes": [ // <-- one on-top layer encapsulating multiple <Board> components
    {
      "lanes": [ // <-- aka columns of single Board
        {
          "id": "PLANNED",
          "title": "Planned Tasks",
          "cards": [
            {
              "id": "Milk",
              "title": "Buy milk",
              "description": "2 Gallons of milk at the Deli store"
            }
          ]
        }
      ]
    }, // <-- the end of the first board
    { // <-- new board
      "lanes": [
        {
          "id": "PLANNED",
          "title": "Planned Tasks",
          "cards": [
            {
              "id": "Milk",
              "title": "Buy milk",
              "description": "2 Gallons of milk at the Deli store"
            }
          ]
        }
      ]
    }
  ]
}
```
Then we map it to please react-trello’s requirements loading in this case two `<Board>` components. But in general it would be a very welcoming feature having native support for multiple horizontal lanes in the JSON data structure instead of adding multiple `<Board>` components. No sure if this is, however, easy to implement...

**Comment by zicodasilva** [comment]
Is there any movement on this feature?
